client/my-sites/site-settings/wrap-settings-form.jsx

```diff
@@ -135,6 +135,13 @@ const wrapSettingsForm = getFormSettings => SettingsForm => {
 			this.props.updateFields( { [ name ]: ! this.props.fields[ name ] } );
 		};
 
+		handleAutosavingToggle = name => () => {
```

Comment:
Side note: if we eventually get rid of all `handleToggle` calls, we can rename `handleAutosavingToggle` to just `handleToggle`, as all toggles would autosave. Right? 😃